### PR TITLE
Fixed Presence

### PR DESCRIPTION
**Pull request by Hanike**

### CODE REVIEW

File: _haikus/The_Presence.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: The Presence
+author: Hanike
+---
+
+There is a presence:<br>
+Silently, the cat watches you<br>
```
Comment (ltagliaferri) [inline _haikus/The_Presence.md]: This line needs 1 fewer syllable

File: _haikus/The_Presence.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+From maddening darkness...<br>
```
Comment (ltagliaferri) [inline _haikus/The_Presence.md]: This final line also needs 1 fewer syllable

File: _haikus/The_Presence.md (hunk 1 above)
Comment (ltagliaferri) [reply _haikus/The_Presence.md]: You can probably remove "you" here to be:
`Silently, the cat watches<br>`

Comment (Hanike) [reply _haikus/The_Presence.md]: Same as the other.. it really seems Hairoku has some counting issues.

Comment (Hanike) [reply _haikus/The_Presence.md]: hairoku-presence-count

File: _haikus/The_Presence.md (hunk 2 above)
Comment (Hanike) [reply _haikus/The_Presence.md]: _Noooo~_ <(ToT)>

**Pull request closed without merging**